retry.go

```diff
@@ -124,14 +127,23 @@ func (r *Retry) postCheck(err error) bool {
 // before the current error is returned.
 // If maxAttempts is 0, then r.Run() will return a nil
 // error on any call.
+// Deprecated: Use MaxAttempts.
 func (r *Retry) Attempts(maxAttempts int) *Retry {
+	return r.MaxAttempts(maxAttempts)
+}
+
+// MaxAttempts sets the maximum amount of retry attempts
+// before the current error is returned.
+// If maxAttempts is 0, then r.Run() will return a nil
+// error on any call.
+func (r *Retry) MaxAttempts(maxAttempts int) *Retry {
```

Comment:
I'd prefer keeping `Attempts` until vgo proliferates. It's not that bad of a name.

Reply:
I think this API has a good potential of becoming standard in the Go community. I feel like a lot of deprecation will scare off developers.

Reply:
Fair enough.